proxy/src/avatar.rs

```diff
@@ -167,10 +176,18 @@ fn generate_emoji(input: &str, usage: Usage) -> Emoji {
             if let Some(s) = EMOJIS.get(ix) {
                 Emoji(s)
             } else {
-                Emoji(EMOJIS_USER[ix - EMOJIS.len()])
+                Emoji(
+                    EMOJIS_USER
+                        .get(ix - EMOJIS.len())
+                        .expect("index of out of range"),
+                )
```

Comment:
Again here, this is a much more verbose way of saying `foo[index]`. It doesn't give us any additional safety.

Reply:
Given that we want to ultimately remove all expects from library code this will force us to think about what we do for the case if this is out of range.

Reply:
I don't see how that will be possible...